public/js/login.js

```diff
@@ -0,0 +1,27 @@
+const username = document.getElementById('username');
+const password = document.getElementById('password');
+const loginButton = document.getElementById('loginButton');
+
+loginButton.addEventListener('click', (e) => {
+  e.preventDefault();
+  const usernameValue = username.value;
+  const passwordValue = password.value;
+  const loginDate = {
+    usernameValue,
+    passwordValue,
+  };
+  fetch('/admin/login', {
+    method: 'POST',
+    headers: { 'content-type': 'application/json' },
+    body: JSON.stringify(loginDate),
+    credentials: 'same-origin',
+  })
+    .then(result => result.json())
+    .then((result) => {
+      if (result.err) return swal('Error !', result.err, 'error');
```

Comment:
I'm gonna need you to do 2 things here:
1. Provide the user a more descriptive error message. i.e. 'Wrong username or password'
2. Split your fetch functions into different files, you seem to have quite a bunch of those, so it will be a lot easier to debug when your fetches are separated from the rest of your code.

Reply:
I sent it from backend 
```
    return response.send(JSON.stringify({ err: 'Wrong username/password' }));
```

Reply:
Continue going through the issues and get back to this one later.